src/coreclr/tools/aot/ILCompiler.RyuJit/Compiler/RyuJitCompilationBuilder.cs

```diff
@@ -25,17 +25,13 @@ public class RyuJitCompilationBuilder : CompilationBuilder
         private string _jitPath;
 
         protected RyuJitCompilationBuilder(CompilerTypeSystemContext context, CompilationModuleGroup group, NodeMangler mangler)
-            : base(context, group, new CoreRTNameMangler(mangler, false))
+            : base(context, group, new NativeAotNameMangler(mangler, false))
         {
         }
 
         public RyuJitCompilationBuilder(CompilerTypeSystemContext context, CompilationModuleGroup group)
-<<<<<<< HEAD
-            : this(context, group, context.Target.IsWindows ? new WindowsNodeMangler() : new UnixNodeMangler())
-=======
             : base(context, group,
                   new NativeAotNameMangler(context.Target.IsWindows ? (NodeMangler)new WindowsNodeMangler() : (NodeMangler)new UnixNodeMangler(), false))
```

Comment:
```suggestion
            : this(context, group, context.Target.IsWindows ? new WindowsNodeMangler() : new UnixNodeMangler())
```
Let's resolve this in our favor since it has simpler code.

Reply:
Done, thanks